### ISSUE TITLE
adopt semantic versioning

### ISSUE THREAD

**Issue by cbm755** [connortannahill/bacoli_py#7]
May I suggest [semver](https://semver.org/)?

  * Basically "x.y.z".
  * Your next version would be either `1.0.2` or `1.1.0`.
  * I'd also recommend bumping the version to `1.0.2+` in git master after the release.

**Comment by connortannahill** [comment]
Good idea! I'll be using semver for the next coming release and future releases.
